src/components/shared/PrizeModal.tsx

```
@@ -183,21 +213,24 @@ export default function PrizeModal({ prize, isOpen, onClose }: PrizeModalProps)
-            <X className="w-4 h-4 sm:w-6 sm:h-6" />
-          </button>
+                {/* Close Button */}
+                <button
+                  onClick={handleClose}
+                  className={`absolute top-3 right-3 sm:top-6 sm:right-6 w-10 h-10 sm:w-12 sm:h-12 backdrop-blur-xl rounded-full flex items-center justify-center transition-all duration-300 theme-transition hover-scale focus-ring ${
+                    theme === "light"
+                      ? "bg-theme-secondary/70 hover:bg-theme-accent text-theme-foreground border border-theme-border"
+                      : "bg-black/70 hover:bg-black/80 text-white border border-white/10"
+                  }`}
+                  style={{ zIndex: 50 }}
+                >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)